README.md

```diff
@@ -20,12 +20,13 @@ yarn storybook
 
 ### Mobile
 
-After running `yarn`, you also need to link assets using:
-
 ```bash
-react-native link
+yarn storybook-native
```

Comment:
Maybe we can make slightly more clear that this step is optional like in official docs https://github.com/storybooks/storybook/tree/next/app/react-native#start-storybook-server-optional .

Also for `Android` we should mention that sometimes port forwarding by running command `adb reverse tcp:7007 tcp:7007` is required to make connection to server work.

Reply:
@JoseDuSV Can you add a commit to this PR with what you suggest?